packages/components/addon/components/hds/tabs/tab.js

```diff
@@ -27,7 +29,10 @@ export default class HdsTabsIndexComponent extends Component {
    * @description Determines if the tab is the selected tab
    */
   get isSelected() {
-    return this.nodeIndex === this.args.selectedTabIndex;
+    return (
+      this.nodeIndex !== undefined &&
+      this.nodeIndex === this.args.selectedTabIndex
+    );
```

Comment:
needed to add a "guard" here to make sure we have a "selected" state only when is not standalone and so the `nodeIndex` is undefined.